corehq/apps/users/templates/users/roles_and_permissions.html

```diff
@@ -164,9 +164,9 @@ <h6 data-bind="visible: !name()">({% trans "No Name" %})</h6>
             </td>
           {% endif %}
           <td class="text-center">
-            <div data-bind="visible: reportPermissions.all(),
-                                            permissionIcon: {
-                                                manage: true
+            <div data-bind="permissionIcon: {
```

Comment:
It may not be necessary to add this to the default page. Specifically, we appear to generally exclude permissions that require a specific subscription level, like this one, since they are confusing for projects that do not have it. Other new permissions like the web apps one are likewise excluded from this page.

Reply:
Let me make sure I understand. I don't think you're saying we should remove the Reports column from this table because it was there before this change. Rather, you're saying that I should just undo this change so the column shows either a check mark, a list of specific reports or empty, but doesn't distinguish the ability to edit from the ability to view. Does that sound right?

Reply:
Ah, that is an excellent catch, and I definitely missed that it was there before. After this change is it possible for a user to be able to be able to create reports, but not view all reports? What is shown if they can only see some of the reports? It looks like the code that lists reports is still there, do they see the eye and the list? That might be crowded. Depending on the answer to the first question, I think the original behavior is fine, or if its not possible to edit but be restricted, having icons for edit and view all, and the list for restricted seems fine as well.

Reply:
It is possible for a user to be able to create reports but not view all reports. In that case, they would see both a check mark and a list of reports. Removing the ability to create reports makes it show just the list, with no eye icon (not really sure why). Giving access to all reports without the ability to create them does show the eye icon with "view only".

I propose that we restore this to the way it was - a check mark when users can access all reports and a list when they can only access specific ones. Since the builtin reports are available with any subscription, this is consistent with your first comment. The ability to access reports is available to all so it makes sense in this table, while the ability to create reports requires pro or higher subscriptions so we won't try to show it in this table.